go.mod

```diff
@@ -12,4 +12,5 @@ require (
 	github.com/sonatype-nexus-community/go-sona-types v0.0.8
 	github.com/sonatype-nexus-community/nancy v1.0.1
 	github.com/spf13/afero v1.3.4
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
There is a v3 of yaml now: https://godoc.org/gopkg.in/yaml.v3

You could use that if you feel DARING. With parsers, etc... it's always nice to be closer to the edge (because the old ones are where they find security vulns, and uh, makes it easier to upgrade when they find one on the new stuff too!)

Reply:
Yeah that's a good idea. Its tough because i tried to match the common library version, but this would easily get out of sync.

Reply:
Go mod itself usually can solve for that (but likely not in this case due to how they created this repo). It does minimum version selection, so if it finds the same library requested (but different versions) and knows it can satisfy the requirement with an older version, etc... it will!

Reply:
Done!